src/EmailCardList/EmailCard/EmailCard.js

```diff
@@ -1,7 +1,16 @@
-const EmailCard = (props) => {
+
+import './DisplayEmailCards.css'
+
+function EmailCard(props) {
+    const handleClick = event => {
+        props.setEmailSearchId(event.currentTarget.id)
+        props.setReadingPanelDisplay(true)
+    }
+  
     return (
-        <div className={(props.read == true ? 'btn-light' : 'btn-secondary') + " btn col-md-12 border-bottom d-flex flex-row justify-content-between p-2"}>
-            <div className="emailContainer">
+        <div className={(props.read == true ? 'btn btn-light col-md-12' : 'btn btn-secondary col-md-12') + "border d-flex flex-row justify-content-between p-2"} id={props.id}
+        onClick={handleClick}>
+           <div className="emailContainer">
```

Comment:
it would make it easier to understand if we rename this class to `emailSnippetContainer`